libraries/AP_InertialSensor/AP_InertialSensor_LSM9DS0.cpp

```diff
@@ -19,7 +19,9 @@
 
 #include <utility>
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_LINUX
 #include <AP_HAL_Linux/GPIO.h>
+#endif
```

Comment:
that line could just be removed as this driver is generic to depend only on AP_HAL/GPIO.h, which is already included from AP_HAL/HAL.H